monkey/infection_monkey/monkey.py

```diff
@@ -215,6 +218,15 @@ def _discover_os(self):
         )
         self._agent_event_queue.publish(event)
 
+    def _discover_hostname(self):
+        event = HostnameDiscoveryEvent(
+            source=self._agent_id,
+            timestamp=time.time(),
+            tags={T1592_ATTACK_TECHNIQUE_TAG},
+            hostname=gethostname(),
```

Comment:
I think we should use `platform.node()`. It seems that is more efficient then `socket.gethostname`.

<img width="678" alt="image" src="https://user-images.githubusercontent.com/15820737/201655053-6849cff1-47b3-4081-9c73-0620f9dc7f3c.png">

Reply:
It's in microseconds which means it won't be noticeable. I don't care about which one we use but another comment in [that thread](https://stackoverflow.com/a/4271873) suggests that `platform.node()` may be unreliable in some (rare) cases.
![image](https://user-images.githubusercontent.com/44770317/201656687-9bc07f92-bd67-4f72-b027-5628342a2002.png)